File: GJOH5/src/task2/PhoneBookMain.java
package task2;

import task2.controller.ManagePhoneBook;

/**
 * Создать телефонный справочник с возможностью импорта и экспорта данных в нескольких форматах
 */
public class PhoneBookMain {
    public static void main(String[] args) {
        ManagePhoneBook managePhoneBook = new ManagePhoneBook();
        managePhoneBook.start();
    }
}

File: GJOH7/phone_book_main.py
from controller.manage_phone_book import ManagePhoneBook


class PhoneBookMain:
    """
    Необходимо реализовать свой проект на каком-то другом языке программирования
    """
    @staticmethod
    def main():
        manage_phone_book = ManagePhoneBook()
        manage_phone_book.start()


if __name__ == "__main__":
    PhoneBookMain.main()
